### ISSUE TITLE
0.5.5 version no longer returns projections.

### ISSUE THREAD

**Issue by shivam-aggarwal** [lucidrains/byol-pytorch#57]
As mentioned in the readme 

```
projection, embedding = learner(imgs, return_embedding = True)
```

Users should be able to get projections and embedding, with 0.5.5 users can no longer do so.
Note : It's happening because of [this](https://github.com/lucidrains/byol-pytorch/blob/master/byol_pytorch/byol_pytorch.py#L152) .

**Comment by lucidrains** [comment]
@shivam-aggarwal Hi Shivam! I've got it fixed in 0.5.6 https://github.com/lucidrains/byol-pytorch/commit/2aa84ee18fafecaf35637da4657f92619e83876d

**Comment by shivam-aggarwal** [comment]
@lucidrains Thank you so much!
Thanks for providing a neat implementation!
